src/main/java/com/bieganski/ox/GameListener.java

```diff
@@ -0,0 +1,5 @@
+package com.bieganski.ox;
+
+public interface GameListener {
```

Comment:
Proszę kwitek tu na GH że te Javadoki nie zostaną zapomniane.

Reply:
Javadoki nie zostaną zapomniane. Jeżeli czegoś nie przeoczyłem teraz powinny być w każdym interfejsie oraz w klasach publicznych.

Reply:
Wygląda na to, że javadocs zostały dodane.